stable/kapacitor/templates/pvc.yaml

```diff
@@ -1,4 +1,5 @@
-{{- if .Values.persistence.enabled }}
```

Comment:
why remove this?

Reply:
not removing adding another test
`{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}`